lib/chef/resource/windows_firewall_rule.rb

```diff
@@ -186,6 +226,7 @@ def firewall_command(cmdlet_type)
           cmd << " -RemotePort '#{new_resource.remote_port.join("', '")}'" if new_resource.remote_port
           cmd << " -Direction '#{new_resource.direction}'" if new_resource.direction
           cmd << " -Protocol '#{new_resource.protocol}'" if new_resource.protocol
+          cmd << " -IcmpType '#{new_resource.icmp_type}'"
```

Comment:
@pschaumburg may be we should add the condition as used for other options ?

Reply:
The condition is not needed for anything. It will always add the IcmpType to the Powershell and do not break anything.